ProvinceMapper/Source/LinkMapper/LinkMapper.cpp

```diff
@@ -38,3 +37,21 @@ void LinkMapper::exportMappings(const std::string& linksFile) const
 	}
 	linkFile.close();
 }
+
+void LinkMapper::deactivateLink()
+{
+	if (activeVersion)
+		activeVersion->deactivateLink();
+}
+
+void LinkMapper::activateLinkByIndex(const int row)
+{
+	if (activeVersion)
+		activeVersion->activateLinkByIndex(row);
+}
+
+void LinkMapper::activateLinkByID(const int ID)
```

Comment:
I believe ID should be unsigned if we don't expect negative IDs, unless you prefer int for negative sentinels.